package/boot/uboot-envtools/files/qualcommax_ipq807x

```diff
@@ -8,6 +8,11 @@ touch /etc/config/ubootenv
 board=$(board_name)
 
 case "$board" in
+spectrum,sax1v1k)
+	part="$(find_mmc_part 0:APPSBLENV)"
+	[ -n "$part" ] && \
```

Comment:
I'm not 100% sure but ain't this supposed to mmcpart

Reply:
$part is temp variable.
Like "prpl,haze", i will change it to $mmcpart

Reply:
fixed and also placed entire case at the end of case blob for better visibility.